internal/handlers/telegram/imgur.go

```diff
@@ -79,5 +79,10 @@ func getImgurLink(tg *Client, tgLink string) string {
 		tg.logger.LogError("Could not retrieve imgur json data:", err)
 	}
 
+	if resp.Data.Deletehash != "" {
```

Comment:
You got it, this is correct. The struct member `Deletehash` is not a pointer, so if it's not in the response JSON then it will be initialized to an empty string.